uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/BAO/Document.php

```diff
@@ -92,6 +92,7 @@ protected static function getDocumentsToClone(DateTime $cutOffDate) {
       $remindMeField => 1,
       'is_deleted' => 0,
       'status_id' => self::STATUS_APPROVED,
+      'is_current_revision' => 1,
```

Comment:
wow, small fix but it was causing lots of redundant data. nice to see this fixed :-)